manage_breast_screening/participants/jinja2/components/participant-details/contact_details.jinja

```diff
@@ -0,0 +1,45 @@
+{% from 'card/macro.jinja' import card %}
+{% from 'summary-list/macro.jinja' import summaryList %}
+
+{% macro contact_details(presented_participant) %}
+  {% set address_html %}
+    {% if presented_participant.address %}
+      {% for line in presented_participant.address.lines %}
+        {{ line }}<br>
+      {% endfor %}
+      {{ presented_participant.address.postcode }}
+    {% endif %}
+  {% endset %}
+  {% call card({
+    "heading": "Contact details"
+  }) %}
+    {{ summaryList({
+      "rows": [
+        {
+          "key": {
+            "text": "Address"
+          },
+          "value": {
+            "html": address_html
+          }
+        },
```

Comment:
The [NHS summary list guidance](https://service-manual.nhs.uk/design-system/components/summary-list) wasn't updated with [**Showing missing information** (GOV.UK)](https://design-system.service.gov.uk/components/summary-list/#showing-missing-information) but what if the address is empty here?

Reply:
Think I'll park this one as well, we've got tickets to add the edit forms, so at that point we will be linking to those.